src/components/Feed/Feed.tsx

```diff
@@ -38,34 +39,26 @@ const Feed = ({ userSentiment, articles, history, updateUserSentiment, storeArti
         image={ article.multimedia.url }
         id={ article.id }
         sentiment={ article.sentiment }
+        topic={ article.topic }
         updateUserSentiment={ updateUserSentiment }
         storeArticle={ storeArticle }
         key={ article.title }
       />
     )
   })
-
-  return (
-    <>
-      <div className="articles-container">
-        <Link to='/history'>
-          <button className='history-btn'>History</button>
-        </Link>
```

Comment:
Can we change the button from text to an icon, something like this (googled 'history icon'):
![Screen Shot 2021-11-08 at 8 38 47 AM](https://user-images.githubusercontent.com/72999840/140782021-55b644f5-3d93-471f-9073-f5f6ed5553d5.png)

Reply:
Ooh, nice. Will do!